test/pylib/scylla_server.py

```diff
@@ -235,7 +238,7 @@ async def cql_is_up(self) -> bool:
                          protocol_version=4,
                          auth_provider=auth) as cluster:
                 with cluster.connect() as session:
-                    session.execute("CREATE KEYSPACE k WITH REPLICATION = {" +
+                    session.execute("CREATE KEYSPACE IF NOT EXISTS k WITH REPLICATION = {" +
```

Comment:
This patch seems to do two things: It increases the CREATE KEYSPACE timeout, and modifies it to CREATE KEYSPACE IF NOT EXISTS, but I don't understand how these things are related to the commit message - or why the IF NOT EXISTS is needed considering you also have a DROP KEYSPACE.

Reply:
If we have a timeout, then we don't know if the keyspace was already created.